cheat_sheets/enumerables.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: cheat_sheet
+topic: Enumerables
+---
+
+### `.times`
+
+Iterates the given block `int` times, passing in values from zero to `int ­- 1`.
+
+##### Structure
+
+```ruby
+number.times do |i|
+  # code
+end
+
+# or
+
+number.times do
+  # code
+end
+```
+
+##### Examples
+
+```ruby
+5.times do |i|
+  puts i
+end
+
+# or
+
+3.times do
+  puts "Hello world!"
+end
+```
+
+### `.each`
+
+Loops through an array, executes the body, returns the original array.
+
+##### Structure
+
+```ruby
+array.each do |value|
+  # code
+end
+```
+
+##### Examples
+
+```ruby
+numbers.each do |number|
+  # code
+end
+```
+
+### `.map`
+
+Creates a new array containing the values returned by the block.
+
+##### Structure
+
+```ruby
+array.map do |value|
+  # code
+end
+```
+
+##### Examples
+
+```ruby
+numbers.map do |number|
+  number + 1
+end
```

Comment:
I think for `map` we need an example where the returned array is actually used (assigned to a variable, for example). Otherwise it's a bit useless...

Reply:
@nerdinand 
I agree, partly... I think it makes sense to assign the result into a variable in this example here. But since it's a cheat sheet I'd go without further explanation, fort that we have the slides, what do you think?